assets/js/modules/analytics-4/datastore/enhanced-measurement.js

```diff
@@ -400,12 +406,21 @@ const baseSelectors = {
 	 * @param {Object} state           Data store's state.
 	 * @param {string} propertyID      The GA4 property ID to check.
 	 * @param {string} webDataStreamID The GA4 web data stream ID to check.
-	 * @return {boolean}               True if `streamEnabled` is on, otherwise false; false if not loaded.
+	 * @return {boolean}               True if `streamEnabled` is on, otherwise false; `undefined` if not loaded.
 	 */
-	isEnhancedMeasurementStreamEnabled( state, propertyID, webDataStreamID ) {
-		return !! state.enhancedMeasurement[ propertyID ]?.[ webDataStreamID ]
-			?.settings?.streamEnabled;
-	},
+	isEnhancedMeasurementStreamEnabled: createRegistrySelector(
+		( select ) => ( state, propertyID, webDataStreamID ) => {
+			const settings = select(
+				MODULES_ANALYTICS_4
+			).getEnhancedMeasurementSettings( propertyID, webDataStreamID );
```

Comment:
This implementation detail was missed when the selector was first added in https://github.com/google/site-kit-wp/issues/7458, but is required.